### DISCUSSION TITLE
collapse the keyboard on android

### DISCUSSION THREAD

**Discussion by skramovskiy** [airsdk/Adobe-Runtime-Support#2589] [General]
Good afternoon
after entering text on android, I want the keyboard to collapse, which command can collapse the android keyboard in AIR

**Comment by ajwfrost** [comment]
Does it work if you remove the focus from the text field? Normally it would collapse the keyboard if you click outside of the text field, so maybe it also works programmatically calling `stage.focus = null;`?
